funfact/loss.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import jax.numpy as np
+
+
+class Loss:
```

Comment:
I think it'll be better to actually split this to three classes, one for each loss function. This one currently looks like a factory method. Do you have some concern regarding separating them?

Reply:
I have split it into 3 classes. If there are loss functions that differ from the default behavior, they can override the `__call__` function from the base class.